lib/transports.js

```diff
@@ -1,14 +1,16 @@
 var events = require('events');
 var util = require('util');
+var assign = require('object-assign');
```

Comment:
Can we drop this dependency? Seems like a silly that can just be replaced by a simple for loop.